docs/release_notes/v0.12.0-alpha.md

```diff
@@ -0,0 +1,58 @@
+# Spice v0.12.0-alpha (Apr 29, 2024)
+
+The v0.11.1-alpha release ...
+
+## Highlights
+
+...
+
+## Contributors
+
+- @phillipleblanc
+- @Jeadie
+- @ewgenius
+- @sgrebnov
+- @y-f-u
+- @lukekim
+- @digadeesh
+- @Sevenannn
+- @gloomweaver
```

Comment:
@ewgenius make sure to update this list and call out david's first contribution.

Reply:
Updated

Reply:
This needs to be updated to include #1204